Gruntfile.js

```diff
@@ -81,6 +82,16 @@ module.exports = function (grunt) {
     config: config,
     loadGruntTasks: {
       pattern: ['grunt-*', '@*/grunt-*', 'gruntify-*', '@*/gruntify-*', 'intern']
+    },
+    preMerge(config, data) {
```

Comment:
Now it is allowing the grunt configs to override their task name, so that config files like `download_selenium.js` can apply to the task named `downloadSelenium`, rather than the task named `download_selenium`